src/engines/outlookcalendar.ts

```diff
@@ -0,0 +1,125 @@
+import * as msal from "@azure/msal-node";
+import * as graph from "@microsoft/microsoft-graph-client";
+import * as dateFns from "date-fns";
+import "isomorphic-fetch";
```

Comment:
Is this needed?

Reply:
Yes, it was part of the Microsoft samples. Apparently, `"@microsoft/microsoft-graph-client"` cannot work without `"isomorphic-fetch"`.

If one tries to remove it, the following error will show up in the terminal:
![image](https://user-images.githubusercontent.com/21250471/156989957-0859872e-7fdd-4ab0-8c35-e9165124b7eb.png)

Reply:
![image](https://user-images.githubusercontent.com/21250471/156991856-92a9a8bc-7233-4917-b7a8-8ee1f63efd48.png)